### PR TITLE
docs: official blog plugin and theme

### PR DESCRIPTION
**Pull request by newsbielt703**
**Summary**

The WIP status has just been removed.

As they are official plugin and theme, It probably should be mentioned in documentation. The content is distilled from README.md, if you have any better idea to introduce them, please leave your comment. 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/theme/default-blog-theme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Default blog theme
```
Comment (kefranabg) [inline packages/docs/docs/theme/default-blog-theme.md]: What about using `Blog Theme` instead? `Default` keyword seems not be correct as it's not the default theme in VuePress, WDYT?

**Comment by kefranabg** [comment]
Shouldn't we add the blog plugin in the plugin section?

File: packages/docs/docs/theme/default-blog-theme.md (hunk 1 above)
Comment (newsbielt703) [reply packages/docs/docs/theme/default-blog-theme.md]: Sure.

**Comment by newsbielt703** [comment]
> Shouldn't we add the blog plugin in the plugin section?

I feel like [the Plugin section](https://deploy-preview-2149--vuepress.netlify.com/plugin/) focus on how to use and write a plugin, so I list them in [the Official Plugins section](https://deploy-preview-2149--vuepress.netlify.com/plugin/official/plugin-active-header-links.html). Besides, I found that it's been mentioned in [examples](https://deploy-preview-2149--vuepress.netlify.com/plugin/#examples).

**Comment by kefranabg** [comment]
Is it ready to merge?

**Comment by newsbielt703** [comment]
I think so

**Pull request merged**